spec/models/account_spec.rb

```diff
@@ -362,42 +362,99 @@
   end
 
   describe 'validations' do
-    it 'has a valid fabricator' do
-      account = Fabricate.build(:account)
-      account.valid?
-      expect(account).to be_valid
+    subject { account }
+    let(:account) { Fabricate.build(:account, username: username, display_name: display_name, note: note, domain: domain) }
+    let(:domain) { nil } # domain = nil means the account is local.
+    let(:username) { 'Gargron' }
+    let(:display_name) { 'Eugen Rochko' }
+    let(:note) { 'Mastodon is a GNU Social-compatible microblogging server.' }
+
+    it { is_expected.to be_valid }
+
+    shared_examples 'check username validation' do
```

Comment:
Unfortunately, I'm not familiar with the gem. But I wanna try to use.
Could you tell me some sample codes you think about this?